portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,4 +43,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json");
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = request.getParameter("comment");
+    messages.add(text);
+    response.setContentType("text/html;");
+    response.getWriter().println(messages);
```

Comment:
This will dump the user to a text only page, a bit odd, but ok for now.
Hopefully the experience can be improved in future assignments.

Reply:
Yes, that's a step in the next few assignments.